sql/src/test/java/io/crate/integrationtests/SysNodeResiliencyIntegrationTest.java

```diff
@@ -77,4 +76,16 @@ public void testTimingOutNode() throws Exception {
             waitNoPendingTasksOnAll();
         }
     }
+
+    /**
+     * Test cluster name check, when path.data does not contain cluster
+     * name as a folder.
+     *
+     * TODO-ES6: Remove this when ES >= 6.0
+     */
+    @Test
+    public void testClusterNameInPathCheckDoesNotFailWithDefaultConfig() throws Exception {
```

Comment:
I think it's okay as there are unit test - I was just questioning if this integration test even makes sense or if it could be removed. But I leave it up to you